app/src/main/java/com/example/caloriecounter/feature/restaurant/ui/categories/CategoryItemComponent.kt

```diff
@@ -0,0 +1,72 @@
+package com.example.caloriecounter.feature.restaurant.ui.categories
+
+import androidx.compose.animation.animateContentSize
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Card
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.example.caloriecounter.R
+import com.example.domain.models.Category
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun CategoryItemComponent(
+    category: Category,
+    modifier: Modifier = Modifier,
+    onClick: (String) -> Unit,
+) {
+    Card(
+        modifier = modifier.fillMaxWidth(),
+        onClick = { onClick(category.name) },
+        elevation = 4.dp,
+    ) {
+        var expanded by rememberSaveable { mutableStateOf(false) }
+        Column {
+            AsyncImage(
+                model = ImageRequest.Builder(LocalContext.current)
+                    .data(category.thumb)
+                    .crossfade(true)
+                    .build(),
+                contentDescription = stringResource(id = R.string.category),
+                contentScale = ContentScale.FillWidth,
+                modifier = Modifier.aspectRatio(16 / 9f)
+            )
+            Spacer(modifier = Modifier.height(16.dp))
+            Text(
+                text = category.name,
+                modifier = Modifier.padding(horizontal = 16.dp),
+                style = MaterialTheme.typography.body1,
+                fontWeight = FontWeight.Bold
```

Comment:
If I'm not wrong, if you have that `fontWeight` set up in the `Type.kt` you do not need to write it here
```suggestion
```

Reply:
Yes, but `body1` usually have `fontWeight = FontWeight.Normal` in my case, same. That's why I added `fontWeight` like this.

Reply:
Maybe better than do MaterialTheme.type.body1.copy(fontWeight = FontWeight.Bold) if it's possible 🤔

Reply:
It's possible. I usually use it when I need to change a couple of qualities.